src/Alpacka.Lib/Instances/MultiMC/MultiMCHandler.cs

```diff
@@ -12,6 +12,7 @@ public class MultiMCHandler : IInstanceHandler
     {
         private readonly Config _config = Config.Load();
         
+        public static readonly string CONFIG_FILE = "instance.cfg";
```

Comment:
Imo static / constant fields should go at the top of the file, definitely before non-static ones.